src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -9530,6 +9548,19 @@ namespace HVACVariableRefrigerantFlow {
 			Tin = Node( OAMixNode ).Temp;
 			Win = Node( OAMixNode ).HumRat;
 		}
+		if ( VRFTU( VRFTUNum ).ATMixerExists ) {
+			// There is an air terminal mixer
+			if ( VRFTU( VRFTUNum ).ATMixerType == ATMixer_InletSide ) {															
+				//Node( VRFTU( VRFTUNum ).ATMixerPriNode ).MassFlowRate = min( Node( VRFTU( VRFTUNum ).ATMixerPriNode ).MassFlowRateMaxAvail, Node( VRFTU( VRFTUNum ).VRFTUInletNodeNum ).MassFlowRate );
+				SimATMixer( VRFTU( VRFTUNum ).ATMixerName, FirstHVACIteration, VRFTU( VRFTUNum ).ATMixerIndex );
+				Tin = Node( VRFTU( VRFTUNum ).ATMixerOutNode ).Temp;
+				Win = Node( VRFTU( VRFTUNum ).ATMixerOutNode ).HumRat;
+			} else {
+				Tin = Node( VRFTU( VRFTUNum ).ATMixerSecNode ).Temp;
+				Win = Node( VRFTU( VRFTUNum ).ATMixerSecNode ).HumRat;
+			}
+		}
+
```

Comment:
The changes in this file are just another reminder of how many _gotchas_ (potential future bugs) there in in the codebase.  Your code is fine @lgu1234, I'm just lamenting for a moment :laughing: